.github/workflows/hf-llm-inference.yml

```
@@ -0,0 +1,23 @@
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: 'Commit to build'
+        required: true
+        default: 'master'
+  push:
+    paths:
+      - "hf-llm-inference/**"
+      - ".github/workflows/hf-llm-inference.yml"
+      - ".github/workflows/build.yml"
+
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yml
+    secrets: inherit
+    with:
+      image-name: hf-llm-inference
+      folder: hf-llm-inference
+      build-args: |
+        COMMIT=${{ github.event.inputs.commit }}
```

This should prevent it from passing `COMMIT=""` on push events
```suggestion
        ${{ inputs.commit && 'COMMIT=' }}${{ inputs.commit }}
```